### ISSUE TITLE
No translation

### ISSUE THREAD

**Issue by muzena** [haecker-felix/gradio#214]
Why there is no available translation in flatpak development version?

**Comment by phw** [comment]
Same here, installed gradio via flatpak, but it isn't localized, even though German translation should be available. Works for me with other Flatpak apps, including my own.

**Comment by julianrichen** [comment]
Doesn't the Locale extension provide translations?

```
flatpak --user install flathub de.haeckerfelix.gradio.Locale
```

**Comment by haecker-felix** [comment]
I can confirm that the translation is not working. I don't know what the problem is. If you know it, you can open a pull request :-)

This is my first project to be translated, so I don't have much experience.

**Comment by phw** [comment]
I have experience with app localization and also did some Flatpak packaging, so I try to debug this a bit.

`de.haeckerfelix.gradio.Locale` is installed, it got installed alongside `de.haeckerfelix.gradio` as it should. When I launch the gradio  flatpak the console reports  

    ** Message: gradio-app.vala:167: Locale dir: /app/share/locale

This looks right. Also when I open a shell into the app with 

    flatpak run --command=sh de.haeckerfelix.gradio 

I can clearly see the localization files in `/app/share/locale`, this looks Ok:

    sh-4.3$ ls /app/share/locale/de/LC_MESSAGES/gradio.mo -lah
    -rw-r--r-- 2 0 0 6,6K Jan  1  1970 /app/share/locale/de/LC_MESSAGES/gradio.mo

Also the initialization of the locales in https://github.com/haecker-felix/gradio/blob/master/src/gradio-app.vala#L156 . I compared everything above to my own https://github.com/phw/peek/blob/master/src/main.vala but don't see a difference. But in my app it works with Flatpak, gradio simple shows default locale texts. Will try to debug further.

Just to make sure I also installed gradio locally and there everything shows up localized.
